antelope_contracts/contracts/erc20/src/erc20.cpp

```diff
@@ -106,7 +283,33 @@ void erc20::handle_evm_transfer(eosio::asset quantity, const std::string& memo)
     call_data.insert(call_data.end(), value_buffer, value_buffer + 32);
 
     call_action call_act(evm_account, {{get_self(), "active"_n}});
-    call_act.send(get_self(), *address_bytes, 0, call_data, evm_gaslimit);
+
+    bytes value_zero; // value of EVM native token (aka EOS)
+    value_zero.resize(32, 0);
+
+    call_act.send(get_self() /*from*/, token.address /*to*/, value_zero /*value*/, call_data /*data*/, evm_gaslimit /*gas_limit*/);
```

Comment:
The `get_self()` here should instead be the variable holding the `eosio.erc2o` name.

Reply:
fixed